src/library/Uno.Material/Styles/Controls/v2/PipsPager.xaml

```diff
@@ -0,0 +1,236 @@
+﻿<ResourceDictionary xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+					xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+					xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+					xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+					xmlns:win="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+					xmlns:not_win="http://uno.ui/not_win"
+					xmlns:muxc="using:Microsoft.UI.Xaml.Controls"
+					mc:Ignorable="d not_win">
+
+	<!-- PipsPager Resources -->
+	
+	<x:Double x:Key="MaterialPipsPagerButtonWidth">20</x:Double>
+	<x:Double x:Key="MaterialPipsPagerButtonHeight">20</x:Double>
+
+	<x:String x:Key="MaterialPipsPagerSelectedGlyph">&#xEA3B;</x:String>
+	<x:String x:Key="MaterialPipsPagerDefaultGlyph">&#xEA3B;</x:String>
+
+	<x:String x:Key="MaterialPipsPagerSelectedGlyphFontSize">8</x:String>
+	<x:String x:Key="MaterialPipsPagerDefaultGlyphFontSize">8</x:String>
+
+	<StaticResource x:Key="MaterialPipsPagerSelectionIndicatorForegroundSelected"
+					ResourceKey="PrimaryBrush" />
```

Comment:
We're trying to move away from aliasing brush resources per control style. It's so we can support Lightweight Styling globally. Aliased resources don't update if you attempt to override the referenced resource

We should instead use `{ThemeResource PrimaryColor}`, etc.